### PR TITLE
ADDED: 2 Quests to QuestDB

### LINKED ISSUE #5334: BUG:  'You've Got Yourself A Deal' Missing From QuestDB
## Description

The SoD quest 'You've Got Yourself A Deal' is not currently in the QuestDB. 

## Wowhead link

https://www.wowhead.com/classic/quest=78297/youve-got-yourself-a-deal

### Discord Bug Report

Quest ID: 78297
Quest Name: You've Got Yourself A Deal
Quest Text: I've signed in the appropriate places, <playername>. Take this back to Ratchet and inform your supplier that the Alliance is happy to do business.
Objective Text: Return the Signed Procurement Contract to Grizzby in Ratchet.
Reward Text: 
Reward XP: 0
Questgiver: Creature-0-5210-0-4646-8383-000002A3F6
Character: Lvl 25 DWARF PRIEST
Player Coords:  [1453]  (79.180, 47.494)
QuestLog: 463, 279, 181, 377, 323, 470, 484, 305, 115, 222, 156, 19, 101, 128, 126, 91, 58, 
Client: 1.15.0 SoD
Questie: v9.2.9
Locale: enUS

## Screenshots

N/a

## Questie & WoW version

v9.2.9
WoW Classic SoD - v1.15.052409

### LINKED ISSUE #5335: BUG: 'Let Me Make You An Offer' Missing From QuestDB
## Description

The SoD quest 'Let Me Make You An Offer' is not currently in the QuestDB. 

## Wowhead link

https://www.wowhead.com/classic/quest=78287/let-me-make-you-an-offer

### Discord Bug Report

Quest ID: 78287
Quest Name: Let Me Make You An Offer
Quest Text: The first run's all packed up, <name>. Take this to Stormwind and get it in front of the higher-ups. Someone who can sign today!
Objective Text: Take the Arms Shipment to Master Wood in Stormwind City.
Reward Text: 
Reward XP: 0
Questgiver: Creature-0-5210-0-4646-8383-000002A3F6
Character: Lvl 25 DWARF PRIEST
Player Coords:  [1453]  (79.180, 47.494)
QuestLog: 463, 279, 181, 377, 323, 470, 484, 305, 115, 222, 156, 19, 101, 128, 126, 91, 58, 
Client: 1.15.0 SoD
Questie: v9.2.9
Locale: enUS

## Screenshots

N/a

## Questie & WoW version

v9.2.9
WoW Classic SoD - v1.15.052409

### PR DESCRIPTION
**Pull request by Sommos**
## Issue references

https://github.com/Questie/Questie/issues/5334
https://github.com/Questie/Questie/issues/5335

Fixes #5334 
Fixes #5335 

## Proposed changes

- ADDED: 'You've Got Yourself A Deal' is now in the QuestDB, and should be accessible to users.
- ADDED: 'Let Me Make You An Offer' is now in the QuestDB, and should be accessible to users.

### CODE REVIEW

File: Database/Corrections/sodQuestFixes.lua [hunk 1]
```diff
@@ -204,6 +204,12 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -415936,
             [questKeys.zoneOrSort] = sortKeys.MAGE,
         },
+        [78287] = {
+            [questKeys.objectivesText] = {"Take the Arms Shipment to Master Wood in Stormwind City."},
+        },
+        [78297] = {
+            [questKeys.objectivesText] = {"Return the Signed Procurement Contract to Grizzby in Ratchet."},
+        },
```
Comment (BreakBB) [inline Database/Corrections/sodQuestFixes.lua:212]:
```suggestion
```

Looks like a mistake to add these to the corrections while the base quest already contains them

Comment (Sommos) [reply Database/Corrections/sodQuestFixes.lua]: Fixed in: https://github.com/Questie/Questie/pull/5336/commits/30b25b0b20d87108fbcfac0b1a2fcd2a868ac914

**Comment by EmilLindegaard** [comment]
Don't mean to be commit-police but i noticed the database does not compile when I updated my master branch because of wrong syntax :)
`[questKeys.preQuestSingle] = {{78287}}` should be `[questKeys.preQuestSingle] = {78287}`

**Comment by BreakBB** [comment]
Aaaaahhh... Thanks @EmilLindegaard 🤦🏻 It is about time #5317 gets merged. I am used to having the pipeline complaining of something does not compile.

**Comment by BreakBB** [comment]
Also wtf, I didn't notice that line in the base file in the first place 😞

**Pull request merged**